recipe/meta.yaml

```
@@ -1,66 +1,75 @@
 {% set name = "tsfresh" %}
-{% set version = "0.18.0" %}
-{% set sha256 = "ebfc1505bc0c0148909a5009190ff49f8ffe4c8c661f7cafa9da6a6ae1a9abf4" %}
+{% set version = "0.21.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9942c2f78523ea12d0403b443293e45d4f1e78f750e5264fabddd2575cd75c43
 
 build:
   number: 0
-  # 2021/10/6: Skip building on osx-arm64 because statsmodels>=0.9.0 isn't available on this platform yet.
-  skip: true  # [arm64]
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
   entry_points:
     - run_tsfresh = tsfresh.scripts.run_tsfresh:main
 
 requirements:
   host:
-    - python >=3.5
     - pip
-    - setuptools_scm
+    - python
     - setuptools
+    - setuptools_scm
+    - pyscaffold >=3.3a0,<4
   run:
-    - python >=3.5
+    - python
     - requests >=2.9.1
     - numpy >=1.15.1
     - pandas >=0.25.0
-    - scipy >=1.2.0
-    - statsmodels >=0.9.0
+    - scipy >=1.2.0  # [py<310]
+    - scipy >=1.14.0  # [py>=310]
+    - statsmodels >=0.13.0
     - patsy >=0.4.1
+    - pywavelets
+    - cloudpickle
     - scikit-learn >=0.22.0
     - tqdm >=4.10.0
-    - dask >=2.9.0
-    - distributed >=2.11.0
-    - matrixprofile >=1.1.10,<2.0.0
     - stumpy >=1.7.2
+    - distributed >=2.11.0
+    - dask[dataframe] >=2.9.0
```

```suggestion
    - dask >=2.9.0
```
conda-build can't parse square brackets [ ]